workflow/executor/docker/docker.go

```diff
@@ -106,27 +106,22 @@ func (d *DockerExecutor) Wait(containerID string) error {
 // killContainers kills a list of containerIDs first with a SIGTERM then with a SIGKILL after a grace period
 func (d *DockerExecutor) Kill(containerIDs []string) error {
 	killArgs := append([]string{"kill", "--signal", "TERM"}, containerIDs...)
-	err := common.RunCommand("docker", killArgs...)
-	if err != nil {
-		return errors.InternalWrapError(err)
-	}
+	// docker kill will return with an error if a container has terminated already, which is not an error in this case.
+	// We therefore ignore any error. docker wait that follows will re-raise any other error with the container.
+	_ := common.RunCommand("docker", killArgs...)
```

Comment:
rather than silently ignore the error, might it be a good idea to just log it?

Reply:
Submitted logging of error.